pkg/config/generator.go

```diff
@@ -33,36 +34,41 @@ const (
 	singleConfigCapabilityKey = "capabilities"
 )
 
+// LoggingOption mutates the `LoggingOptions` object
+type LoggingOption func(loggingOptions *types.LoggingOptions)
+
 // Option mutates the `conf` object
 type Option func(conf *MultusConf)
 
 // MultusConf holds the multus configuration, and persists it to disk
 type MultusConf struct {
-	BinDir                   string          `json:"binDir,omitempty"`
-	Capabilities             map[string]bool `json:"capabilities,omitempty"`
-	CNIVersion               string          `json:"cniVersion"`
-	Delegates                []interface{}   `json:"delegates"`
-	LogFile                  string          `json:"logFile,omitempty"`
-	LogLevel                 string          `json:"logLevel,omitempty"`
-	LogToStderr              bool            `json:"logToStderr,omitempty"`
-	Kubeconfig               string          `json:"kubeconfig"`
-	Name                     string          `json:"name"`
-	NamespaceIsolation       bool            `json:"namespaceIsolation,omitempty"`
-	RawNonIsolatedNamespaces string          `json:"globalNamespaces,omitempty"`
-	ReadinessIndicatorFile   string          `json:"readinessindicatorfile,omitempty"`
-	Type                     string          `json:"type"`
+	BinDir                   string                `json:"binDir,omitempty"`
+	Capabilities             map[string]bool       `json:"capabilities,omitempty"`
+	CNIVersion               string                `json:"cniVersion"`
+	Delegates                []interface{}         `json:"delegates"`
+	LogFile                  string                `json:"logFile,omitempty"`
+	LogLevel                 string                `json:"logLevel,omitempty"`
+	LogToStderr              bool                  `json:"logToStderr,omitempty"`
+	LoggingOptions           *types.LoggingOptions `json:"loggingOptions,omitempty"`
```

Comment:
How about to use 'LogOptions' instead of 'Logging' to make it consistent with 'LogFile' 'LogLevel' and so on?

Reply:
Agree, Done